styleguide.md

```diff
@@ -0,0 +1,180 @@
+# Modelica Specification style guide
+
+This is the style guide for the Modelica Specification document.
+
+
+## Document format and tool chain
+
+The source format of the document is LaTeX, and the source is processed by both pdfLaTeX and LaTeXML.
+It is good to be aware of the LaTeXML implications, but for most contributors and contributions is is considered sufficient to only check that the pdfLaTeX output looks good, the idea being that potential problems should be spotted in the pull request review process.
+
+
+## Source code formatting
+
+The document is in ongoing transition to _one sentence per line_ source code formatting.
+This means that any modified or new text should have each sentence alone on a single physical line in the source file.
+Once we have the physical line breaks in the correct places, the diffs of future changes will become clean and easy to grasp.
+
+When indenting the contents of a LaTeX environment, an indentation of 2 spaces used.
```

Comment:
Should be "is used" or something similar.

Reply:
Yes, added "is".  Resolving.